pkg/bootkube/status.go

```diff
@@ -34,13 +34,16 @@ func WaitUntilPodsRunning(pods []string, timeout time.Duration) error {
 	if err := wait.Poll(5*time.Second, timeout, sc.AllRunning); err != nil {
 		return fmt.Errorf("error while checking pod status: %v", err)
 	}
+
+	fmt.Println("All self-hosted control plane components successfully started")
 	return nil
 }
 
 type statusController struct {
-	client    clientset.Interface
-	podStore  cache.Store
-	watchPods []string
+	client        clientset.Interface
+	podStore      cache.Store
+	watchPods     []string
+	lastPodPhases map[string]v1.PodPhase
```

Comment:
It needs to persist between the polling of the `func (s *statusController) AllRunning() (bool, error)` function. 

I'm not sure if it belongs here myself, but, I think other options get more complicated too. Open to suggestions.